www/js/controllers.js

```diff
@@ -144,7 +144,7 @@ angular.module('starter.controllers', [])
 
 .controller('InfoCtrl', function($scope, $http, $stateParams) {
 
-  $http.get('https://hikehk.herokuapp.com/trails/'+$stateParams.id).success(function(data, status, xhr){
+  $http.get('http://localhost:3000/trails/'+$stateParams.id).success(function(data, status, xhr){
```

Comment:
You should make this a global variable, so you can change everything url at the same time.

In the future, you can change it to an environment variable, so that it will just work.